frontend/src/screens/learner.tsx

```diff
@@ -142,6 +156,34 @@ const Filters: React.FC<FiltersProps> = ({ studies, filter, setFilter }) => {
         </Box >
     )
 }
+
+interface AllSubjectsProps extends FiltersProps {
+    onSelect(study: ParticipantStudy): void
+}
+
+const AllSubjects: React.FC<AllSubjectsProps> = ({
+    onSelect,
+    filter,
+    setFilter,
+    studies,
+}) => {
+    if (useIsMobileDevice()) {
+        return (
+            <>
+                {StudyTopicTagIDs.map((tag) => (
```

Comment:
Normally, initial cap indicates a constructor.

Reply:
yep, good point, thanks!

Reply:
actually I'll fix in one of the other stacked prs so they don't conflict.